ducktape/tests/loader.py

```diff
@@ -106,7 +107,7 @@ def load(self, test_discovery_symbols):
             # packing tests into bins based on the least full bin at the time.
             raw_results = _requests_session.get(self.historical_report).json()["results"]
             time_results = {r['test_id']: r['run_time_seconds'] for r in raw_results}
-            avg_result_time = sum(time_results.itervalues()) / len(time_results)
+            avg_result_time = sum(itervalues(time_results)) / len(time_results)
```

Comment:
possible divide by zero?

Reply:
This shouldn't be possible since this block only makes sense with a result file, which would have to have contents (or would likely be malformed for this use anyway).